spec/models/miq_product_feature_spec.rb

```diff
@@ -16,16 +14,34 @@
     )
   end
 
+  it "is properly configured" do
+    everything = YAML.load_file(described_class.feature_yaml)
+    required_keys = [:identifier]
+    possible_keys = required_keys + [:name, :feature_type, :description, :children, :hidden, :protected]
```

Comment:
@jrafanie makes sense to me. I was hoping for feedback when I wrote that because I sort of inferred that this was so but wasn't sure. But this is indeed the wrong place to define this knowledge if it's correct